easybuild/easyconfigs/f/FFTW/FFTW_3.3.4-p2.patch

```diff
@@ -0,0 +1,871 @@
+#https://www-user.tu-chemnitz.de/~potts/workgroup/pippig/software.php.en#scripts
+#Within these scripts we apply the following patches that have been submitted to FFTW but are not yet included in FFTW-3.3.4:
+#We add two more global transposition algorithms to the planner.
+#We patch file mpi/transpose-pairwise.c in order to fix a double free bug.
```

Comment:
please mention that this was fleshed out from `install_fftw-3.3.4_gcc.sh`, and that it's required for PFFT

maybe we could also give the patch a more meaningful name?